pkg/upgrade/upgrades/upgrades.go

```diff
@@ -240,6 +240,13 @@ var upgrades = []upgradebase.Upgrade{
 		upgrade.NoPrecondition,
 		backfillJobTypeColumn,
 	),
+	upgrade.NewTenantUpgrade(
+		"creat virtual column indexes_usage based on (statistics-> 'statistics'-> 'indexes') with index "+
```

Comment:
nit: `s/creat/create/`, and remove spaces in `(statistics-> 'statistics'-> 'indexes')`